retrofit2-clients/src/test/java/com/palantir/remoting3/retrofit2/TestService.java

```diff
@@ -50,9 +50,6 @@
     @GET("relative")
     Call<String> getRelative();
 
-    @GET("/absolute")
```

Comment:
Absolute URLs don't work with the redirect mechanism. I suspect we don't really need absolute URLs anyway for any of our RPC?